### PR TITLE
Update whamg again

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: tests/modules/nf-core/whamg/test.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
...
-  command: nextflow run ./tests/modules/nf-core/whamg -entry test_whamg_bam -c ./tests/config/nextflow.config  -c ./tests/modules/nf-core/whamg/nextflow.config
+  command: nextflow run ./tests/modules/nf-core/whamg -entry test_whamg_bam -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/whamg/nextflow.config
   tags:
     - whamg
   files:
     - path: output/whamg/test.vcf.gz
+      md5sum: 35c622fb7f1c8306fdadd11cc9fa833a
+    - path: output/whamg/test.vcf.gz.tbi
+      md5sum: e7180bb953d2bd657c420a5f76a7164d
+    - path: output/whamg/versions.yml
```
Comment (pinin4fjords) [inline tests/modules/nf-core/whamg/test.yml:10]: Do we need this?

**Review by pinin4fjords — CHANGES_REQUESTED** [review]

File: modules/nf-core/whamg/main.nf [hunk 2]
```diff
@@ -2,37 +2,40 @@ process WHAMG {
...
     path(fasta)
     path(fasta_fai)
 
     output:
-    tuple val(meta), path("*.vcf.gz"), emit: vcf
-    path("*.txt")                    , emit: graph, optional: true
-    path "versions.yml"              , emit: versions
+    tuple val(meta), path("*.vcf.gz")       , emit: vcf
+    tuple val(meta), path("*.vcf.gz.tbi")   , emit: tbi
+    path("*.txt")                           , emit: graph, optional: true
```
[the remark this answers is not in the document]
Comment (nvnieuwk) [reply modules/nf-core/whamg/main.nf:18]: Yes you're right! Good spot!

File: tests/modules/nf-core/whamg/test.yml (hunk 1 above)
Comment (nvnieuwk) [reply tests/modules/nf-core/whamg/test.yml:10]: What do you mean exactly?

Comment (pinin4fjords) [reply tests/modules/nf-core/whamg/test.yml:10]: Sorry, probably naive question- but is it conventional to have the versions file in the test.yml? Or, actually, is this different to the regular versions file?

Comment (nvnieuwk) [reply tests/modules/nf-core/whamg/test.yml:10]: Ah yes! The tests need to check for the presence of the `versions.yml` file only (this is also automatically generated using `nf-core modules create-test-yml`)

Comment (pinin4fjords) [reply tests/modules/nf-core/whamg/test.yml:10]: I guess what I'm getting at is that output/whamg/versions.yml isn't in the output files (though versions.yml is), so is it important to check its presence in the output?

Comment (nvnieuwk) [reply tests/modules/nf-core/whamg/test.yml:10]: I don't really follow your reasoning here sorry, `versions.yml` is in the `output/whamg` folder

Comment (pinin4fjords) [reply tests/modules/nf-core/whamg/test.yml:10]: Sorry, just me having a dim moment, never mind

**Review by pinin4fjords — APPROVED** [review]

File: tests/modules/nf-core/whamg/test.yml (hunk 1 above)
Comment (nvnieuwk) [reply tests/modules/nf-core/whamg/test.yml:10]: No problem, happens to the best of us :p

**Pull request merged**